foundation/blockchain/database/block.go

```diff
@@ -241,7 +241,7 @@ func (b Block) ValidateBlock(previousBlock Block, stateRoot string, evHandler fu
 
 		parentTime := time.Unix(int64(previousBlock.Header.TimeStamp), 0)
 		blockTime := time.Unix(int64(b.Header.TimeStamp), 0)
-		if !blockTime.After(parentTime) {
+		if blockTime.Before(parentTime) {
```

Comment:
Nice Change! Anytime we can remove fuzzy or negative logic, the better.

Reply:
:)